framework/test/jest/unit/specs/controller/channels/child_process_channel.spec.js

```diff
@@ -382,7 +391,7 @@ describe('ChildProcessChannel Channel', () => {
 	describe('#_rejectWhenAnySocketFailsToBind', () => {
 		beforeEach(() => childProcessChannel.registerToBus(socketsPath));
 
-		it('should reject if any of the sockets receive an "error" event', () =>
+		it('should reject if any of the sockets receive an "error" event', async () =>
```

Comment:
here function is async but .rejects wasn't awaited.

Reply:
removed async as it's a one-liner with implicit return